### ISSUE TITLE
Feature Request: Limits for parallel processes of chrome

### ISSUE THREAD

**Issue by juev** [ArchiveBox/ArchiveBox#994]
## Type 

 - [ ] General question or discussion
 - [ ] Propose a brand new feature
 - [x] Request modification of existing behavior or design

## What is the problem that your feature request solves

In this time Archive box will run many chrome processes. How many processes we don't know. We have situation, when chrome processes eats all system resources.

Some backgrounds:
- ArchiveBox running in docker
- I imported ~500 links
- I see a lot processes in system monitor

<img width="1440" alt="archivebox" src="https://user-images.githubusercontent.com/114635/176170127-faa0e5d8-7ee2-4508-8c44-8d5b0676f716.png">

## Describe the ideal specific solution you'd want, and whether it fits into any broader scope of changes

We have separate options, which limits how many links will processed in parallel. It will limit number of running chrome processes.

## What hacks or alternative solutions have you tried to solve the problem?

Just restart docker compose. :(

## How badly do you want this new feature?

 - [ ] It's an urgent deal-breaker, I can't live without it
 - [x] It's important to add it in the near-mid term future
 - [ ] It would be nice to have eventually

---

 - [ ] I'm willing to contribute [dev time](https://github.com/ArchiveBox/ArchiveBox#archivebox-development) / [money](https://github.com/sponsors/pirate) to fix this issue
 - [x] I like ArchiveBox so far / would recommend it to a friend
 - [ ] I've had a lot of difficulty getting ArchiveBox set up

**Comment by pirate** [comment]
Let me know if you're still seeing this issue in version >=`0.7.2` or `:dev`. Chrome headless is much improved since the time this ticket was open.
